<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberWidget</class>
 <widget class="QWidget" name="MemberWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>691</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-color:blue;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>You may also like</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New arrivals</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#C3C3C3;
</string>
    </property>
    <property name="text">
     <string>J. R. R. Tolkien</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#C3C3C3;
</string>
    </property>
    <property name="text">
     <string>Victor Hugo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#C3C3C3;
</string>
    </property>
    <property name="text">
     <string>Alexandre Dumas</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>270</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#C3C3C3;
</string>
    </property>
    <property name="text">
     <string>Leo Tolstoy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="hobbitButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>The Hobbit</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="lesMisButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Les Miserables</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="blackTulipButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>The Black Tulip</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="warButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>War and Peace</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>550</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#C3C3C3;
</string>
    </property>
    <property name="text">
     <string>George Orwell</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#C3C3C3;
</string>
    </property>
    <property name="text">
     <string>Frank Herbert</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>560</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#C3C3C3;
</string>
    </property>
    <property name="text">
     <string>J. R. R. Tolkien</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>550</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#C3C3C3;
</string>
    </property>
    <property name="text">
     <string>Leo Tolstoy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="lotrButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>520</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>The Lord ofâ€¨the Rings</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="duneButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dune</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="annaButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>520</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Anna Karenina</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="orwellButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>520</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1984</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white
</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white
</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white
</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: white
</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white
</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white
</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white
</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_8" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white
</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue
</string>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>752</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="browsePushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yourBooksPushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Your books</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
